dnsdbq.c

```diff
@@ -1314,10 +1316,33 @@ print_rateval(FILE *outstream, const char *key, const struct rateval *tp) {
 	fputc('\n', outstream);
 }
 
-/* write_info -- assumes that reader contains the complete json block
+/* print_burstrate -- output formatter for burst_size and burst_window ratevals
  */
 static void
-write_info(reader_t reader) {
+print_burstrate(FILE *outstream, const char *key, const struct rateval *tp_size, const struct rateval *tp_window) {
+	/* if unspecified, output nothing, not even the key name. */
+	if (tp_size->rk == rk_naught || tp_window->rk == rk_naught)
+		return;
+
+	assert(tp_size->rk == rk_int);
+	assert(tp_window->rk == rk_int);
+
+	u_long b_w = tp_window->as_int;
+	u_long b_s = tp_size->as_int;
+
+	fprintf(outstream, "\t%s: ", key);
+	if (b_w == 3600) fprintf(outstream, "%lu per hour", b_s);
```

Comment:
Please correct the formatting here. the if or else shouldn't be on the same line as the statement that it is conditionalizing.

Reply:
Fixed.